src/pages/App.tsx

```diff
@@ -146,6 +149,11 @@ export default function App() {
     setScrolledState(false)
   }, [pathname])
 
+  const [searchParams] = useSearchParams()
+  useEffect(() => {
+    if (searchParams.has('disableNFTs')) setShouldDisableNFTRoutes(true)
```

Comment:
you should probably do an explicit comparison here to = true since if users pass in `disableNFTs=false` this would be wrong